packages/snap-client/src/Client/transforms/searchResponse.ts

```diff
@@ -386,13 +386,17 @@ transformSearchResponse.sorting = (response: searchResponseType) => {
 
 transformSearchResponse.merchandising = (response: searchResponseType) => {
 	const merchandising = response?.merchandising;
-
+	const transformedMerchandising: any = merchandising;
```

Comment:
Should be typing to use for this.

Reply:
I think we should also refactor this entire block so that only what we define are included as properties on merchandising - not passing through anything else.

Eg:
```
transformSearchResponse.merchandising = (response: searchResponseType) => {
	const merchandising = response?.merchandising;

	const transformedMerchandising = {
		redirect: merchandising?.redirect || '',
		content: merchandising.content?.length ? merchandising.content : {},
		campaigns: merchandising?.triggeredCampaigns || [],
	};

	return {
		merchandising: transformedMerchandising,
	};
};
```